SpaceDome/src/booster.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+#include <iostream>
+#include <vector>
+#include <string>
+#include <memory>
+#include <AssimpLoader.h>
+#include "assimp/Importer.hpp"
+
+#include "mesh.h"
+#include "utility.h"
+#include "player.h"
+#include "bullets.h"
+
+class Player;
+
+/**
+ * @class Booster 
+ */
+class Booster {
+
+public:
+    /**
+     * @brief Constructor for Booster class. 
+     */
+    Booster() { position = generateBoosterPos(); 
+        
+    }
+    /**
+     * @brief Activates booster, is done in game.cpp when a player is close enough to a booster. 
```

Comment:
Det bör framgå vad en överlagrad funktion förväntas göra. Bör nog framgå att den kan modifiera spelaren som skickas in.
Att den används i game.cpp är för specifikt - det kan man enkelt söka upp om man vill. Detta gäller generellt för dina kommentarer om funktioner.